language/evm/move-to-yul/src/events.rs

```diff
@@ -186,6 +186,163 @@ impl EventSignature {
     }
 }
 
+/// Generate emit functions for send_<name>
+pub(crate) fn define_emit_fun_for_send(
```

Comment:
Would be nice to gate those functionalities under a `async_move` compilation flag.

Reply:
Added a gate to generate this function only when the contract is annotated with #[actor]